ATGMediaBrowser/MediaBrowser/MediaBrowserViewController.swift

```diff
@@ -192,6 +192,18 @@ public class MediaBrowserViewController: UIViewController {
             contentViews.forEach({ $0.updateTransform() })
         }
     }
+    /// Variable to set title in media browser
+    public override var title: String? {
+        didSet {
+            titleLabel.text = title
+        }
+    }
+    /// Variable to hide/show title control in media browser.
+    public var shouldShowTitle: Bool = true {
```

Comment:
Let's keep the title visibility to false as default please. This will allow for backward compatibility, and will not suddenly show a title on updating the framework for users.